figures/helpers.py

```diff
@@ -110,6 +110,10 @@ def prev_day(val):
     return days_from(val, -1)
 
 
+def days_in_month(month_for):
+    return calendar.monthrange(month_for.year, month_for.month)[1]
```

Comment:
No biggie, better than having the magical constant `1`.

```suggestion
    """
    Calculate days in a certain month.
    """
    _from_day, to_day = calendar.monthrange(month_for.year, month_for.month)[1]
    return to_day
```

Reply:
@OmarIthawi Sure, we'll do that, but we need to drop the `[1]` from the call you suggest, and the first value is the day of the week of the first day of the month. This is the new function:

```
def days_in_month(month_for):
    _, num_days_in_month = calendar.monthrange(month_for.year, month_for.month)
    return num_days_in_month
```